torchvision/prototype/transforms/_container.py

```diff
@@ -38,6 +38,7 @@ def extra_repr(self) -> str:
 
 class RandomChoice(Transform):
     def __init__(self, *transforms: Transform) -> None:
+        # This method should receive optionally a list of probabilities and sample transforms proportionally.
```

Comment:
To be able to fine tune which transform is chosen? Like 30% transform1 and 70% transform2?